qmarkdowntextedit.cpp

```diff
@@ -106,7 +106,7 @@ void QMarkdownTextEdit::setHighlightingEnabled(bool enabled) {
         _highlighter->setDocument(enabled ? document() : Q_NULLPTR);
     }
 
-    if (enabled && _highlighter) {
```

Comment:
Now you removed `_highlighter` instead of `enabled`. :laughing:  And `_highlighter->setDocument(enabled ? document() : Q_NULLPTR);` will not be reached if `enabled == false`

Reply:
check now lol. I just messed up the commit.